raster/raster-layer-url/src/main/java/com/esri/samples/raster_layer_url/RasterLayerURLSample.java

```diff
@@ -63,8 +68,8 @@ public void start(Stage stage) {
         }
       });
 
-      // create a map with dark canvas vector basemap
-      ArcGISMap map = new ArcGISMap(Basemap.createDarkGrayCanvasVector());
+      // create a map with dark gray basemap style
```

Comment:
Just said I won't comment, but this one definitely needs either `a` or `the`, doesn't matter which 😁 
```suggestion
      // create a map with a dark gray basemap style
```